valohai_yaml/objs/pipelines/execution_node.py

```diff
@@ -42,3 +43,13 @@ def get_parameter_defaults(self) -> Dict[str, Any]:
             in self.override['parameters']
             if 'default' in param
         }
+
+    def get_input_defaults(self) -> Dict[str, Any]:
```

Comment:
Wouldn't this have to be
```suggestion
    def get_input_defaults(self) -> Dict[str, List[str]]:
```